components/support/nimbus-cli/src/main.rs

```diff
@@ -105,24 +107,48 @@ impl TryFrom<&Cli> for LaunchableApp {
             _ => None,
         };
 
+        // Scheme for deeplinks.
+        let scheme = match app {
+            "fenix" => Some(match channel {
+                "developer" => "fenix-dev",
+                "nightly" => "fenix-nightly",
+                "beta" => "fenix-beta",
+                "release" => "fenix",
+                _ => unreachable!(),
+            }),
+            "firefox_ios" => Some(match channel {
+                "developer" => "fennec",
+                "beta" => "firefox-beta",
+                "release" => "firefox-internal",
```

Comment:
why is it called `"firefox-internal"`?? is that really what they call their release channel? 🤔

Reply:
I started typing an explanation, but then realised it should be a comment in the code. I've added a comment.